lib/rules/indent.js

```diff
@@ -616,6 +617,7 @@ module.exports = {
             MemberExpression: 1,
             ArrayExpression: 1,
             ObjectExpression: 1,
+            ImportDeclaration: 1,
```

Comment:
I think that should be handled under a separate option in a separate PR, myself.